### DISCUSSION TITLE
Why we need network throttling to measure UI performance

### DISCUSSION THREAD

**Discussion by SivasubramanianV** [GoogleChrome/lighthouse#15923] [Q&A] [answered]
Hi, this question might seem basic however i would like to understand if lighthouse is an UI performance metric tool, and it calculates performance of how UI elements are loaded in client browser after the resources have been retrieved from server, why do we need network throttling? Or we are also measuring how effective UI elements and js pages are retrieved from server so that it impacts the time to load. Please provide some clarification on this.

**Comment by adamraine** [comment]
I'm not sure what you are asking. Network load times affect performance. Lighthouse adds performance throttling is added to emulate the imperfect network conditions that real users will experience.

**Reply by SivasubramanianV** [reply]
@adamraine yes, it might have been a basic question even but i wanted to know is there an aspect of lighthouse metric that does not depend on network throttling.
I was thinking given all resources were retrieved from server or cdn for 2 similiar pages, 
From that point, Can there be a difference in lighthouse score between those 2 pages, based on how the html/jsx page was designed.
 One aspect could be if one application had defined proper sizes for its images in the pages, the CLS would be good and score will increase.

**Reply by adamraine** [reply, marked as answer]
> yes, it might have been a basic question even but i wanted to know is there an aspect of lighthouse metric that does not depend on network throttling

I would expect the TBT and CLS metrics to be *less sensitive* to network throttling (they are typically more sensitive to CPU throttling and other factors). That being said, I can certainly imagine scenarios where changes to the network traffic can indirectly impact TBT and CLS so we can't make any guarantees.

> I was thinking given all resources were retrieved from server or cdn for 2 similiar pages,
From that point, Can there be a difference in lighthouse score between those 2 pages, based on how the html/jsx page was designed.
One aspect could be if one application had defined proper sizes for its images in the pages, the CLS would be good and score will increase.

If the pages have different content then the Lighthouse scores can certainly be different. Even if the pages and servers were identical they can be subject to [variance](https://github.com/GoogleChrome/lighthouse/blob/main/docs/variability.md).

**Reply by SivasubramanianV** [reply]
Thanks Adam
